File: mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/identifierdialog/ClientIdentifierCreationListener.java
package com.mifos.mifosxdroid.dialogfragments.identifierdialog;

import com.mifos.objects.noncore.Identifier;

/**
 * Created by Tarun on 07-08-17.
 */

public interface ClientIdentifierCreationListener {

    void onClientIdentifierCreationSuccess(Identifier identifier);

    void onClientIdentifierCreationFailure(String errorMessage);
}

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/identifierdialog/ClientIdentifierCreationListener.kt
package com.mifos.mifosxdroid.dialogfragments.identifierdialog

import com.mifos.core.objects.noncore.Identifier

/**
 * Created by Tarun on 07-08-17.
 */
interface ClientIdentifierCreationListener {
    fun onClientIdentifierCreationSuccess(identifier: Identifier)

    fun onClientIdentifierCreationFailure(errorMessage: String)
}